src/azure-openapi-validator/rules/DescriptionMustNotBeNodeName.ts

```diff
@@ -0,0 +1,29 @@
+import { rules } from "../rule";
+
+rules.push({
+  id: "M2065",
+  name: "DescriptionMustNotBeNodeName",
+  severity: "error",
+  category: ["RPCViolation"],
+
+  appliesTo_JsonQuery: "$..*[?(@.description)]",
+  run: function* (doc, node, path) {
+    let msg: string = "Description must not match the name of the node it is supposed to describe";
```

Comment:
Do we need `.` and can we push this message as part of rules.push json? like below:

```json5
 id: "M2065",
 name: "DescriptionMustNotBeNodeName",
 severity: "error",
 category: ["RPCViolation"],
 msg: ".....",
```